release notes/v0.45.0.md

```diff
@@ -0,0 +1,169 @@
+- [#3099](https://github.com/grafana/k6/pull/3099) replace "breached" with "crossed" in logs around thresholds. Thanks to @MattDodsonEnglish 🙇.
+- [#3102](https://github.com/grafana/k6/pull/3102) Better error message when SharedArray constructor is provided with an async function. This is not supported, but the original message wasn't very clear.
+- [#3089](https://github.com/grafana/k6/pull/3089) Add Software Bill of Materials (SBOM) reports to k6 releases. Thanks to @SadFaceSmith 🙇.
+
+## Bug fixes
+
+* [#3058](https://github.com/grafana/k6/pull/3058) fix repetitive `the`. Thank you, @cuishuang 🙇.
+
+## Maintenance and internal improvements
+
+- [#2091](https://github.com/grafana/k6/pull/2991) Refactor JS modules system so that is usable in tests. Which allowed the enablement of the tc39 tests for modules [#3040](https://github.com/grafana/k6/pull/3040).
+- [#3025](https://github.com/grafana/k6/pull/3025) Internally stop referring to afero and use an internal package to do all file system interaction. That package still uses afero.
+- [#3036](https://github.com/grafana/k6/pull/3036) and [#3053](https://github.com/grafana/k6/pull/3053) Add options to `scenarios` for usage by browser module. 
+- [#3064](https://github.com/grafana/k6/pull/3064), [#3070](https://github.com/grafana/k6/pull/3070),  [#3075](https://github.com/grafana/k6/pull/3075) and [#3106](https://github.com/grafana/k6/pull/3106) Go dependancies updates. 
+- [#3067](https://github.com/grafana/k6/pull/3067) Add method to retrive all registered metrics.
+- [#3068](https://github.com/grafana/k6/pull/3068) Add metric Sink constructor.
+- [#3078](https://github.com/grafana/k6/pull/3078) Pinning to Alpine 3.17 in Docker builder image. Thank you, @arukiidou 🙇.
+- [#3086](https://github.com/grafana/k6/pull/3086) Fix downloading `.golangci.yml` for PRs from forks. 
+- [#3088](https://github.com/grafana/k6/pull/3088) Make TestDNSResolver less flaky. 
+- [#3094](https://github.com/grafana/k6/pull/3094) Fix example from the run command. Thanks to @rocktwotj 🙇.
+- [#3095](https://github.com/grafana/k6/pull/3095) Maintenance update of `.golangci.yml`.
+- [#3103](https://github.com/grafana/k6/pull/3103) Fix lint and logical issues in `k6/data` module tests.
+- [#3045](https://github.com/grafana/k6/pull/3045), [#3049](https://github.com/grafana/k6/pull/3049), [#3073](https://github.com/grafana/k6/pull/3073) and [#3044](https://github.com/grafana/k6/pull/3044) New issues are now automatically assigned to maintainers, to improve response time on issues. Both new issue and new PR assignments are now not using external actions.
+## Roadmap
+
+There is now a [public Roadmap document](https://github.com/grafana/k6/blob/master/ROADMAP.md) in the main repo added in [#3028](https://github.com/grafana/k6/pull/3028)
+
+The idea is that we will keep it updated and current or future users will be able to look at it to see on what bigger projects the team is working on.
+
+### Cloud output v2
+ TODO
```

Comment:
We should not mention it, we are not adding any improvement for the end user, so if we really want to mention them then let's add it to `Maintenance and internal improvements` section.

Reply:
This is in the future/roadmap section. And it is TODO specifically so we discuss this.

Reply:
Or do you mean that even when we release it - we should not be putting it in the release notes?

Reply:
>This is in the future/roadmap section. And it is TODO specifically so we discuss this.

So would you like to announce it as an upcoming improvement? Why do you think is it important?

>Or do you mean that even when we release it - we should not be putting it in the release notes?

No, I meant in the current release notes. I expect it will be one more item in the `Maintenance and internal improvements` section when we will release it.

Reply:
> Why do you think is it important?

You did do quite a lot of work on it, and it will be quite a big change even if (hopefully) no users manages to notice it.

Reply:
Agree with Mihail. I think this is important to mention, even though it's not quite ready for public release. But k6 Cloud users will be interested to know about the changes.

So maybe we don't need to link each individual GH issue, but mention that we're working on a Cloud output overhaul, describing all the improvements, and that it will be ready by next release.